### ISSUE TITLE
23801324: iTunes Connect warns about AppIcon size but file is correct

### ISSUE THREAD

**Issue by openradar-mirror** [lionheart/openradar-mirror#6925]
#### Description

Summary:
When uploading a new build, Application Loader warns that an App Icon file has the wrong size, however this file was created by xcodebuild from a AppIcon in an asset catalog.

Steps to Reproduce:
Upload a .ipa file build by xcodebuild

Expected Results:
The build passes validation

Actual Results:
Application Loader reports the following error: WARN: WARNING ITMS-90393: "Invalid Icon Name. The watch application 'XXXX' contains an invalid icon file name 'AppIcon44x44@2x.png'. Make sure that icon file names follow the pattern "*<dimension>@<scale>x.png" and they match the required dimensions. Please visit https://developer.apple.com/watchkit/ for more information."

Regression:
The submitted .ipa does contain two watch apps, a watchOS 1 and a watchOS 2 app. Both contain a file called AppIcon44x44@2x.png with a size of 88x88 pixels.

Notes:
The Apple ID of the application in question is XXXX.
## 

Product Version: 
Created: 2015-12-08T14:30:27.348020
Originated: 2015-12-08T00:00:00
Open Radar Link: http://www.openradar.me/23801324

**Comment by fbartolom** [comment]
I have exactly the same error and submitted a bug report to Apple. I actually found one 44x44@2x.png file in the archive, possibly created from the 88x88 item in the assets.

**Comment by mrezk** [comment]
I found a solution for this warning on the [following thread](https://forums.developer.apple.com/message/92565#92565) from Apple Developer Forums:

> kimyabu
> Dec 19, 2015 7:54 AM
> 
> I have finally solved it. Error message is so vague and obscure. I added that icon before since Xcode didn't allow me to upload WatchOS build without that icon. However that icon is actually supported by WatchOS 1 only.So if you only target WatchOS 2 and want to get rid of that error, go to your AppIcon and select only "watchOS All Versions" and remove tick box next to "watchOS 1.0 Specific". When you do this, your AppIcon44x44@2x.png will be unassigned. Get rid of that file by deleting it and your project will build without a warning.

**Comment by fbartolom** [comment]
That's right, of course I could not get it by submitting the binary, but the slot has actually disappeared.
